public/img/what_to_do/what-to-do-4.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="42px" height="42px" viewBox="0 0 42 42" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 51.1 (57501) - http://www.bohemiancoding.com/sketch -->
+    <title>4</title>
+    <desc>Created with Sketch.</desc>
+    <defs>
+        <linearGradient x1="100%" y1="0%" x2="0%" y2="100%" id="linearGradient-1">
+            <stop stop-color="#18BAFF" offset="0%"></stop>
+            <stop stop-color="#0A84FF" offset="100%"></stop>
+        </linearGradient>
+    </defs>
+    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="What-to-do" transform="translate(-721.000000, -97.000000)">
+            <g id="Numbering">
+                <g transform="translate(722.000000, 98.000000)">
+                    <circle id="Oval" stroke="url(#linearGradient-1)" stroke-width="2" cx="20" cy="20" r="20"></circle>
+                    <text id="1" font-family="FiraSans-Medium, Fira Sans" font-size="20" font-weight="400" fill="#0A84FF">
+                        <tspan x="15" y="26">4</tspan>
+                    </text>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
All 4 of these SVGs are the same, except for the `<tspan x="15" y="26">4</tspan>` and `<title>4</title>` tags. Not sure if there is a way to pass a variable into an SVG, or set via content via CSS or something.

Reply:
We could! So we could do to the 'What To Do' steps what was done to the 'Password Advice' Tips and pass a variable to one cleaned up inline SVG. I am not aware of a way to style SVGs that are embedded with <img> tags from the CSS though.

Reply:
I pulled this off in 88cc010e17ba3b56383274446dd5a8093802cd4c by creating a handlebars partial - `what_to_do_svg.hbs` - that accepts a `number` argument and renders the SVG inline with the appropriate number.

Goodbye 4 duplicate images!

Reply:
Currently lost in the abyss of https://www.w3.org/TR/SVGParamPrimer/ and trying to figure out if that's a real thing.

Not to say your solution isn't beautiful (if not SVG verbose in the HTML markup), but something like this seems fancy: `<object type="image/svg+xml" data="what_to_do.svg?number=1" />`

Of course, with great fanciness probably comes limited browser support.

Reply:
It looks like `<object data="what_to_do.svg?number=1" />` would trigger the same HTTP requests that `<img>` tags do. The nice thing about the inline SVG approach is it cuts out the HTTP requests, and the "bloat" of the SVG-in-HTML is hidden by the handlebars partials.